src/commcare_cloud/ansible/roles/haproxy/templates/service.j2

```diff
@@ -34,7 +34,6 @@ backend {{ item.service.haproxy_service_name }}-back
     stick-table type ip size 256k expire 30m
 {% if request_option == "http" %}
     option forwardfor
-    option httplog
```

Comment:
[WARNING] 274/092350 (6199) : parsing [/etc/haproxy/haproxy.cfg:57] : backend 'couchdb2-back' : 'option httplog' directive is ignored in backends.